### PR TITLE
Added an out-of-scope section to the explainer

### PR DESCRIPTION
**Pull request by iherman**
Making the out-of-scope part in the charter is, editorially, probably not feasible.

The section was inspired by the mail of @danbri, although shortened significantly.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/lds-wg-charter/pull/82.html" title="Last updated on May 14, 2021, 9:24 AM UTC (e6a2478)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/lds-wg-charter/82/4a88284...e6a2478.html" title="Last updated on May 14, 2021, 9:24 AM UTC (e6a2478)">Diff</a>

### CODE REVIEW

**Comment by danbri** [comment]
Looks good to me

File: explainer.html [hunk 1]
```diff
@@ -225,6 +226,16 @@ <h2>Defining an RDF Dataset Hash and/or Signing RDF Datasets</h2>
     </div>
   </section>
 
+  <section id="out-of-scope">
+    <h2>Out-of-scope Issues for the Working Group Scope</h2>
```
Comment (pchampin) [inline explainer.html:230]:
```suggestion
    <h2>Out-of-scope Issues for the Working Group</h2>
```
it feels like one "scope" too many...

**Review by pchampin — APPROVED** [review]
Fine with me. Something that is not explicitly spelled out is the notion of "social meaning of a signature", which is also out-of-scope. My understanding is that it is implied by "Authenticity and trust issues", but maybe it is worth being even more explicit about it?

Unfortunately, I am not at all sure on how to word it...

**Review by dlongley — APPROVED** [review]

**Comment by iherman** [comment]
@pchampin, good point. I have added a remark to the end of the first paragraph. It now reads as:

> These are all but a few examples where the complexities around the usage of Linked Data/RDF are real, and may affect the usage patterns of signed RDF content and what the social meaning on those signatures are.

The next paragraph starts by saying that all those are out-of-scope:-)

**Review by peacekeeper — APPROVED** [review]

**Comment by iherman** [comment]
Merging to make further discussions easier and avoiding merge hells. The addition is an incremental addition to what was there, ie, further improvement (if necessary) can be done without problems.

**Comment by kdenhartog** [comment]
I'm not sure I understand the implications of this well enough to agree or disagree with this. One of the footguns we've commonly encountered is that due to the nature of the contexts resolvability and contents properties within the JSON-LD may be getting dropped during the signature or verification process. Is the intent of this statement to place that work out of scope?

**Comment by iherman** [comment]
If I understand well your question, probably neither yes or no, but more closely to 'yes', ie, it may be out of scope indeed.

The specs are all defined on the abstract data type level, ie, on the RDF graph/dataset side. The specs also define a vocabulary to express the results. The specs are not aimed to make any normative statement on the specific serializations, and JSON-LD is but one of these. I would expect the documents (or additional notes) may address these types of issues, too, but not as part of the normative sections. The "out-of-scope" sections in the charter (or, in this case, the explainer document) concentrate on the normative parts of the future specifications.

Does this help?

**Comment by kdenhartog** [comment]
Yeah, thanks for the response. I found the issue which precipitated this PR (#83) , so I've moved the discussion over there.

**Pull request merged**